src/vs/platform/quickinput/browser/quickAccess.ts

```diff
@@ -36,16 +36,17 @@ export class QuickAccessController extends Disposable implements IQuickAccessCon
 		return this.doShowOrPick(value, true, options);
 	}
 
-	show(value = '', options?: IQuickAccessOptions): void {
-		this.doShowOrPick(value, false, options);
+	async show(value = '', options?: IQuickAccessOptions): Promise<void> {
```

Comment:
I wonder if we could just keep the method return `void` and not force all the other changes? How important is it to let callers `await` the `show`?